docs/pages/versions/unversioned/sdk/updates.mdx

```diff
@@ -10,102 +10,128 @@ platforms: ['android', 'ios', 'tvos']
 
-<ConfigReactNative>
+If using [app config](/workflow/configuration/) for configuration, this module can be configured by setting at least the following app config keys:
+- [`updates.url`](/versions/latest/config/app/#updates) - a URL of a remote service implementing the [Expo Updates protocol](/technical-specs/expo-updates-1/)
+- [`runtimeVersion`](/versions/latest/config/app/#runtimeversion) - a [runtime version](/eas-update/runtime-versions/)
 
-Learn how to configure the native projects in the [installation instructions in the `expo-updates` repository](https://github.com/expo/expo/tree/main/packages/expo-updates#installation-in-bare-react-native-projects).
+The remote service must implement the [Expo Updates protocol](/technical-specs/expo-updates-1/). [EAS Update](/eas-update/introduction) implements this protocol for you, and it is also possible to use this module with a custom server.
 
-</ConfigReactNative>
+<BoxLink
+  title="Custom Expo Updates Server"
+  description="Example implementation of a custom server and an app using that server"
+  href="https://github.com/expo/custom-expo-updates-server"
+  Icon={GithubIcon}
+/>
 
-## Usage
+</Collapsible>
 
-Most of the methods and constants in this module can only be used or tested in release mode. In debug builds, the default behavior is to always load the latest JavaScript from a development server. It is possible to [build a debug version of your app with the same updates behavior as a release build](/eas-update/debug-advanced/#debugging-of-native-code-while-loading-the-app-through-expo-updates). Such an app will not open the latest JavaScript from your development server &mdash; it will load published updates just as a release build does. This may be useful for debugging the behavior of your app when it is not connected to a development server.
+### Additional configuration options
 
-**To test manual updates in the Expo Go app**, run [`eas update`](/eas-update/introduction) and then open the published version of your app with Expo Go.
+These are build-time configuration options that control the behavior of the module. For most apps, these configuration values can be set in the [app config](/workflow/configuration/) under the [`updates` field](/versions/latest/config/app/#updates).
 
-**To test manual updates with standalone apps**, you can create a [**.apk**](/build-reference/apk) or a [simulator build](/build-reference/simulators), or make a release build locally with `npx expo run:android --variant release` and `npx expo run:ios --configuration Release` (you don't need to submit this build to the store to test).
+You can also configure the module in your app's native project files if your project does not use Continuous Native Generation. It is also possible to override the configuration from native code.
```

Comment:
@ide - going to change back to "library" everywhere, re: https://github.com/expo/expo/pull/29275#discussion_r1623266005

not a problem at all, just want to raise in case we want to change the style guide

Reply:
Ok I withdraw my suggestion to use "package"... will remember to use "library" going forward 😄

Reply:
Let's follow the style guide!